### ISSUE TITLE
[Bug] export animation as avi crashes GRASS

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass#835]
Since exporting an animation as gif is not working because of #827, I tried exporting as avi. GRASS crashes completely while exporting, I get this in the terminal:

```
 [1]+  Stopped                 grass78
```

and all windows frozen. My animation is 20 maps long plus a vector layer for boundaries.

**System description (please complete the following information):**
```
GRASS version: 7.8.3                                                            
Code revision: exported                                                         
Build date: 2020-06-26                                                          
Build platform: x86_64-redhat-linux-gnu                                         
GDAL: 3.0.4                                                                     
PROJ: 6.3.2                                                                     
GEOS: 3.8.1                                                                     
SQLite: 3.32.3                                                                  
Python: 3.8.3                                                                   
wxPython: 4.0.7                                                                 
Platform: Linux-5.7.8-200.fc32.x86_64-x86_64-with-glibc2.2.5
```

**Comment by petrasovaa** [comment]
I can't reproduce this with, it works for me. Does this happen only for large animations?

**Comment by veroandreo** [comment]
It fails also with 5 maps here... I don't think 20 is a large number of maps anyway (and map size is pretty small, i.e., 100x100 cells). Any idea what else could it be? Dependencies?

**Comment by petrasovaa** [comment]
You need ffmpeg, the GUI should warn you if you don't have it.

**Comment by veroandreo** [comment]
I have it... and indeed, there was not such warning... :thinking:

**Comment by tmszi** [comment]
I can confirm this bug.

**To reproduce**:

1. Launch wxGUI `grass --gui`
2. Launch Animation tool from Layer Manager **Modules** page (tab) only
3. Try export animation file as avi

* if `g.gui.animation` is  launched from emulator terminal or Console page (tab) , bug doesn't occur

**Comment by petrasovaa** [comment]
Should be fixed by #840.
